pkgs/servers/etcd/3.4.nix

```diff
@@ -26,6 +26,8 @@ buildGoModule rec {
     install -Dm755 bin/* bin/functional/cmd/* -t $out/bin
   '';
 
+  passthru.tests = { inherit (nixosTests) etcd etcd-cluster; };
```

Comment:
Pointless to add this as it is a no op. Only `etcd` is tested, not the versioned packages.

The test will also fail for >= 3.4.

```suggestion
```

Reply:
What about other versions?

I can give a look at tests and update it too.

Reply:
Also, is it worth leaving 3.5.1 as the default version?